build.sbt

```diff
@@ -40,46 +40,46 @@ val commonsCodecVersion = "1.15"
 val commonsCompressVersion = "1.21"
 val datastoreV1ProtoClientVersion = "2.9.0"
 val flinkVersion = "1.15.0"
-val googleClientsVersion = "1.32.1"
-val googleCloudDatastoreVersion = "0.93.10"
-val googleOauthClientVersion = "1.33.3"
+val googleClientsVersion = "2.0.0"
+val googleOauthClientVersion = "1.34.1"
 val guavaVersion = "31.1-jre"
-val hadoopVersion = "2.10.1"
+val hadoopVersion = "2.10.2"
 val httpCoreVersion = "4.4.14"
-val jacksonVersion = "2.13.0"
+val jacksonVersion = "2.13.3"
 val javaxAnnotationApiVersion = "1.3.2"
-val nettyTcNativeVersion = "2.0.47.Final"
-val nettyVersion = "4.1.52.Final"
+val nettyTcNativeVersion = "2.0.52.Final"
+val nettyVersion = "4.1.77.Final"
 val sparkVersion = "3.1.2"
-val threetenbpVersion = "1.4.4" // from google-cloud-platform/build.gradle
 // dependent versions
-val googleApiServicesBigQueryVersion = s"v2-rev20211129-$googleClientsVersion"
-val googleApiServicesDataflowVersion = s"v1b3-rev20210818-$googleClientsVersion"
-val googleApiServicesPubsubVersion = s"v1-rev20211130-$googleClientsVersion"
-val googleApiServicesStorageVersion = s"v1-rev20211201-$googleClientsVersion"
+val googleApiServicesBigQueryVersion = s"v2-rev20220827-$googleClientsVersion"
+val googleApiServicesDataflowVersion = s"v1b3-rev20220812-$googleClientsVersion"
+val googleApiServicesPubsubVersion = s"v1-rev20220829-$googleClientsVersion"
+val googleApiServicesStorageVersion = s"v1-rev20220705-$googleClientsVersion"
 
 // check versions from libraries-bom
 // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/index.html
```

Comment:
Do you think it is worth to specify in comments that "com.google.cloud:libraries-bom:26.1.1" is used? Otherwise it is not directly clear

Reply:
I know. I wanted to avoid doing like in beam build file, where they also have a link which is very often not updated and points to the wrong version.